resources/lang/en/posts.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+return [
+    'create'        => [
+        'title' => 'New Post',
+    ],
+    'fields'        => [
+        'name'          => 'Name',
+        'parent'        => 'Parent Entity',
+        'position'      => 'Position',
+    ],
+    'placeholders'  => [
+        'name'  => 'Name of the post',
```

Comment:
Same here, we have all these translations already in all languages of kanka in `entities/notes` translation file :)

Reply:
These are auto generated by the code.